addons/music/fnc_playMusic.sqf

```diff
@@ -0,0 +1,48 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_playMusic
+Description:
+    A function used to play music. 
+Parameters:
+    1: CLASS Class to play
+    2: INT Position to play from (default: 0)
+    3: BOOL allowed to interupt already playing music (default: true)
+Example:
+    (begin example)
+    _bool = ["LeadTrack01_F_Bootcamp", 30, true] call CBA_fnc_playMusic;
+    (end example)
+Returns:
+    BOOL
+Author:
+    Fishy
+---------------------------------------------------------------------------- */
+#include "script_component.hpp"
+private ["_config","_data","_canPlay","_duration","_return"];
+params [["_className",""],["_time",'name'],["_overWrite", true]];
```

Comment:
* "" and '' 
* you should add the check for the right variabletype too
* `_time` has the wrong default value